caom2/caom2/plane.py

```diff
@@ -91,6 +91,8 @@
            'PlaneURI', 'DataQuality', 'Metrics', 'Provenance', 'Position', 
            'Energy', 'Polarization', 'Time']
 
+_OBSCORE = "http://www.ivoa.net/std/ObsCore"
+_CAOM = "http://www.opencadc.org/caom2/DataProductType"
```

Comment:
Now that these are outside of the DataProductType class, perhaps they should be renamed?  Maybe something like:
_OBSCORE_VOCAB_NS
_CAOM_DPT_VOCAB_NS